index.html

```diff
@@ -4703,7 +4575,7 @@ <h3>Verify Securing Mechanism</h3>
 <var>inputBytes</var> as an input parameter, parses the value as JSON into
 an object, removes the `proof` property, and returns the result as an object.
 If JSON parsing fails, a <a href="#PARSING_ERROR">PARSING_ERROR</a> is appended
```

Comment:
(@jyasskin has probably more experience in using infra, so he should be the arbiter on whether the following comment is justified or not...)

In step 5.1) below, it says, that _result.document_ is set to the output of this algorithm. In the introduction of this section, _result.document_ is set to an (infra) _map_. This means that the _extractDocument_ should produce, formally, a _map_ and not an "object" as it says here.

Fortunately, that is also covered by the infra standard: it defines the ["parse JSON bytes to an infra value"](https://infra.spec.whatwg.org/#parse-json-bytes-to-an-infra-value) step that formally provides that "bridge" between the input bytes into what is needed in infra. I think this is what should be used here.

Note that the same comment applies to 3.1.1)

Reply:
That's correct, but I'm also hoping to replace _extractDocument_ entirely with https://github.com/w3c/vc-data-integrity/pull/227, so you might not want to spend time fine-tuning this bit of the algorithm.

Reply:
@iherman I'm going to wait and land @jyasskin's PR https://github.com/w3c/vc-data-integrity/pull/227 and then come back and clean up the interfaces here.